ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/util/CommandParser.java

```diff
@@ -61,8 +61,8 @@
       Pattern.CASE_INSENSITIVE);
   private static final Pattern UNSET_PROPERTY = Pattern.compile(
       "\\s*UNSET\\s+'((?:[^']*|(?:''))*)'\\s*;\\s*", Pattern.CASE_INSENSITIVE);
-  private static final Pattern DROP_CONNECTOR =
-      Pattern.compile("\\s*DROP\\s+CONNECTOR\\s+(.*?)\\s*;\\s*", Pattern.CASE_INSENSITIVE);
+  private static final Pattern DROP_CONNECTOR = Pattern.compile(
+      "\\s*DROP\\s+CONNECTOR\\s+(IF\\s+EXISTS\\s+)?(.*?)\\s*;\\s*", Pattern.CASE_INSENSITIVE);
```

Comment:
For my own understanding, should there be a similar update for a regex for the create connector change?

Reply:
Because `CREATE CONNECTOR` statements have so many components in it, we just use the ksql parser to extract the class name/properties etc instead of using regex.